Makefile

```
@@ -0,0 +1,13 @@
+PHONY: default deps dev
+
+default: dev
+
+deps:
+	git submodule init
+	git submodule update
+
+dev:
+	hugo server --minify --theme book
+
+compile:
+	 hugo -d docs --minify --theme book
```

I'd prefer if the output directory was called `dist` or something along those lines. We might want to create a directory called `docs` in future that contains information about how to work on this repository.

```suggestion
	 hugo -d dist --minify --theme book
```

---

`docs` (or `dist` once renamed) should also be `.gitignore`'d from the repository.

---

This line is also inconsistently indented compared to other lines.